gradle.properties

```diff
@@ -9,24 +9,24 @@
 # org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
 #
 # When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. More details, visit
+# This option should only be used with decoupled projects. For more details, visit
```

Comment:
The only required change seems like this in this PR to me. Please do not hesitate to explain your point for other changes.